pkgs/os-specific/linux/zenmonitor/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, pkgconfig, gtk3, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  name = "zenmonitor";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "ocerman";
+    repo = "zenmonitor";
+    rev = "v${version}";
+    sha256 = "10pgw2i8gn8grg6gg43kcipj11lipnd9fyw31jnwrzgwyv4z0w0w";
+  };
+
+  buildInputs = [ gtk3 ];
+  nativeBuildInputs = [ pkgconfig wrapGAppsHook ];
+
+  # TODO: use the default installPhase in the next release (>1.2)
+  installPhase = ''
+    install zenmonitor -Dt $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Monitoring software for AMD Zen-based CPUs";
+    homepage = https://github.com/ocerman/zenmonitor;
+    license = licenses.mit;
+    platforms = [ "i686-linux" "x86_64-linux" ];
```

Comment:
can this be?
```suggestion
    platforms = platforms.linux;
```

Reply:
I think `x86_64-linux` is most accurate, as Zen is AMD64 only.

Reply:
Installing this package really only makes sense on x86.